recipes/jigsaw/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "JIGSAW" %}
+{% set version = "0.9.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/dengwirda/jigsaw/archive/v{{ version}}.tar.gz
+  sha256: 75b2a7454b8fc3cc7d95475f915101d35ba0e434630f4668601772320f016ed2
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+
+test:
+  source_files:
+    - example.jig
+    - geo/bunny.msh
+    - out/temp.txt
+  commands:
+    - jigsaw example.jig
+
+about:
+  home: https://github.com/dengwirda/jigsaw
+  license: custom
```

Comment:
Is something like this okay?

Reply:
In general yes provided the licence text is included with `licence_file` and it states the package can be freely distributed.

I checked on the gitter chat and someone from the core team confirmed that licence is fine. 👍